tensornetwork/backends/numpy/decompositions.py

```diff
@@ -122,3 +122,22 @@ def rq(
   r = np.reshape(r, list(left_dims) + [center_dim])
   q = np.reshape(q, [center_dim] + list(right_dims))
   return r, q
+
+
+def cholesky(
+    np: Any,
+    tensor: Tensor,
+    pivot_axis: int
+) -> Tuple[Tensor, Tensor]:
+  """
+  Computes the Cholesky decomposition of a tensor
+
+  See tensornetwork.backends.tensorflow.decompositions for details.
+  """
+  left_dims = np.shape(tensor)[:pivot_axis]
+  right_dims = np.shape(tensor)[pivot_axis:]
+  tensor = np.reshape(tensor,
+                      [np.reduce_prod(left_dims),
```

Comment:
`np` is the `numpy` module. `numpy` does not have `reduce_prod`, that's a `tensorflow` routine. This only passes the test because in `decompositions_test.py` you are erroneously passing the `tensorflow` module instead of the `numpy` module to `numpy.decompositions.cholesky`. pls fix this